uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -210,4 +202,51 @@ public void activateAdmin() {
             Assert.fail(e.getMessage());
         }
     }
+
+    /**
+     * @return returns the button of date in the settings screen.
+     */
+    private UiObject changeDateButton() {
+        if (android.os.Build.VERSION.SDK_INT == 28)
+            return UiAutomatorUtils.obtainUiObjectWithText("Set date");
+
+        return UiAutomatorUtils.obtainEnabledUiObjectWithExactText("Date");
+    }
+
+    /**
+     * @param Text Text for which we need to find button in the UI.
+     * @return returns the button of the matched text in the scrollable UI.
+     */
+    private UiObject obtainButtonInScrollable(final String Text) {
+        if (android.os.Build.VERSION.SDK_INT == 28)
+            return UiAutomatorUtils.obtainChildInScrollable(
+                    "com.android.settings:id/list",
+                    Text
+            );
+
+        return UiAutomatorUtils.obtainChildInScrollable(
+                "com.android.settings:id/recycler_view",
+                Text
+        );
+    }
+
+    /**
```

Comment:
Either provide complete javaodoc.....or just remove it as this is a private method